client/transaction-pool/graph/src/pool.rs

```diff
@@ -149,23 +149,27 @@ impl<B: ChainApi> Pool<B> {
 	}
 
 	/// Imports a bunch of unverified extrinsics to the pool
-	pub async fn submit_at<T>(
+	pub async fn submit_at(
 		&self,
 		at: &BlockId<B::Block>,
 		source: TransactionSource,
-		xts: T,
-		force: bool,
-	) -> Result<Vec<Result<ExtrinsicHash<B>, B::Error>>, B::Error> where
-		T: IntoIterator<Item=ExtrinsicFor<B>>,
-	{
-		let validated_pool = self.validated_pool.clone();
+		xts: impl IntoIterator<Item=ExtrinsicFor<B>>,
+	) -> Result<Vec<Result<ExtrinsicHash<B>, B::Error>>, B::Error> {
 		let xts = xts.into_iter().map(|xt| (source, xt));
-		self.verify(at, xts, force)
-			.map(move |validated_transactions| validated_transactions
-				.map(|validated_transactions| validated_pool.submit(validated_transactions
-					.into_iter()
-					.map(|(_, tx)| tx))))
-			.await
+		let validated_transactions = self.verify(at, xts, true).await?;
```

Comment:
Can we change the `bool` flag to an enum, I just realised you flipped the meaning and was confused with all the `false->true` changes. Since it's a super important flag I think it's worth making it more descriptive.